docs/rules/valid-expect.md

```diff
@@ -3,6 +3,9 @@
 💼 This rule is enabled in the ✅ `recommended`
 [config](https://github.com/jest-community/eslint-plugin-jest/blob/main/README.md#shareable-configurations).
 
+🔧 This rule is automatically fixable by the
```

Comment:
can we have it say that "some patterns are fixable" or something like that? specifically now, only if the function is already async

Reply:
I attempted to fix it with commit 90028bf. What do you think?

Reply:
fwiw it seems currently the docs generator doesn't support customizing this in anyway, so I think what you've done is fine for now - @bmish might be something worth adding; even just the ability to explicitly name particular rules as "partially fixable" in the config which would case this string to include with "in some cases" suffix